src/cli/README.md

```diff
@@ -33,11 +33,11 @@ OpenThread test scripts use this CLI to execute test cases.
 * [stop](#stop)
 * [whitelist](#whitelist)
 
-## OpenThread Command Details
+## OpenThread command details
 
 ### channel
 
-Get the IEEE 802.15.4 Channel value.
```

Comment:
As an American English rule, only proper words should have first caps where the first letter of the word capitalized, such as "Thread" and "Jonathan". (Compare this to German where all nouns are capitalized.) We can create exceptions, but we have to recognize them as such. It is customary to use full caps for variables, such as PAN ID (also listed as "panid"). I don't think terms qualify for capitalization if you are referring to terms such as "channel" and "masterkey", but again, we can specify our preferences. If you point me to dev docs that you think are ideal, I will write up their rules, and we can adopt those. I am not trying to make this more complicated. I am trying to standardize our usage because this lets us set the rules. Nest wants to develop a "Developer voice" but no one is doing is, so let's start now.